docs/sources/whatsnew/whats-new-in-v8-0.md

```diff
@@ -163,6 +162,57 @@ These improvements affect any subsequent data updates or interactions, including
 
 The following data source updates are included with this Grafana release.
 
+#### Azure Monitor data source
+
+Grafana 8.0 includes some major changes and new functionality to make using and configuring the Azure Monitor data source easier and simpler, and to pave the way for future improvements.
```

Comment:
Add a link in this paragraph about how the Azure Monitor data source topic was updated as a result of these changes. 

Also, keep in mind that the point of the What's new doc is to tell the user WHAT has changed and to provide a link to the docs with the details of HOW it has changed. Try to keep this content sparse, especially if it is likely to get stale in future releases.

Reply:
Had a chat with @joshhunt, I will create a separate PR for the What's New content. He will refine this PR next week.